src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingSession.ts

```diff
@@ -596,6 +596,9 @@ export class ChatEditingSession extends Disposable implements IChatEditingSessio
 		// ensure that the edits are processed sequentially
 		this._sequencer.queue(() => this._acceptTextEdits(resource, textEdits, isLastEdits, responseModel));
 	}
+	acceptNotebookEdits(_resource: URI, _edits: ICellEditReplaceOperation[], _isLastEdits: boolean, _responseModel: IChatResponseModel): void {
+		//
```

Comment:
This is an implementation detail, hence left out for now.

It makes sense to have `IModifiedFileEntry.atteptAgentEdits` to support `NotebookEdits` as well, but not sure we wanted to bring that change in here yet. Either way that will require more work hence left that out of this PR.

Reply:
Yeah, I agree. Let's leave it empty and figure out post PR